Side Bar.sublime-menu

```diff
@@ -1,6 +1,11 @@
 [
-    {"caption": "-"},
-    { "caption": "PEP8: Format Code", "command": "auto_pep8_file", "args": {"paths": [], "preview": false} },
-    { "caption": "PEP8: Preview Code", "command": "auto_pep8_file", "args": {"paths": [], "preview": true} },
-    {"caption": "-"}
+    { 
+    	"caption": "AutoPEP8", 
+    	"id": "autopep8",
+    	"children":
+        [
+		    { "caption": "Format Code", "command": "auto_pep8_file", "args": {"paths": [], "preview": false} },
+		    { "caption": "Preview Code", "command": "auto_pep8_file", "args": {"paths": [], "preview": true} },
```

Comment:
Please remove last comma: "Unable to parse value: Trailing comma before closing bracket at Packages/AutoPEP8/Side Bar.sublime-menu:8:5"